### ISSUE TITLE
Change "edit mode" button icon to mouse pointer

### ISSUE THREAD

**Issue by JoelWiebe** [encorelab/ck-board#55]
There is a button that changes from navigation mode (where the user can click on and drag the canvas) to an edit mode (where the user can click and drag the notes). This button is on the right side and switches between a hand and a pencil icon, respectively. 

This behaviour may be more clear if we switch the "pencil icon" to a "mouse pointer icon".

**Comment by JoelWiebe** [comment]
Hi @sadmanca, I was thinking that the "hand" icon would be displayed when we are in the mode where you can pan around the canvas by grabbing the canvas and the "four-arrow" icon could be displayed when clicking drags the notes around.

Is it possible to add the functionality that I can still open a note from "pan" mode? I just don't want to be accidentally sliding notes around when I'm trying to pan across the canvas.

What do you think about switching back to having the hand icon when in pan mode and adding the ability to open notes in pan mode? (Therefore the modes aren't so much pan vs edit but pan vs move -- as both modes could allow for editing).

**Comment by JoelWiebe** [comment]
P.S. Alternatively, if we can't open a note from pan-mode, maybe we can double click in pan-mode to get back to move-mode, what do you think?

**Comment by sadmanca** [comment]
I think the current icon setup is slightly confusing because of how the pan/edit modes work (i.e. you can't edit posts while panning mode is on, and you can't pan while being able to edit posts), and that you don't really know that the button is a toggle until you click it. 

Maybe instead of a toggle button, we could have 2 buttons visible side-by-side, with the "four-arrow" icon representing the edit/move posts mode and the "hand" icon representing the pan mode? The currently active mode could be coloured in, while the inactive mode could be gray. I think this could more clearly convey what mode is currently active and how to switch to other mode as well. 

I've added a mockup of what this could look like below. Let me know what you think.

![Frame 1](https://user-images.githubusercontent.com/41028402/133932389-d465c6b5-068c-4db6-a776-f73640ba92af.png)

**Comment by sadmanca** [comment]
I agree that it would be more intuitive if clicking or dragging on a post allows you to edit or move it, while dragging on the canvas allows you to pan around. 

Of course, like you mentioned, accidentally sliding around posts could be annoying, and could also be an obstacle if we wanted to implement a feature where clicking and dragging on the canvas allows you to select multiple posts (e.g. for assigning to a bucket, deleting, changing colour). 

We could potentially tie pan mode to be active while a key (e.g. spacebar) is held down, which would be visually represented by the cursor being a "hand" icon and, if we proceed with a 2 button layout, the pan mode button being shown as active while the spacebar is held down. This could probably be done with minimal changes to our code for the canvas component, since we're not changing the underlying functionality of how the modes work.

**Comment by Ammar-T** [comment]
I think we should display the icon that represents the current mode we are in. So, if we're in pan, show the panning icon. 

Also, I agree with being able to edit posts while panning. I can create a new issue and add that functionality.

**Comment by JoelWiebe** [comment]
@Ammar-T, displaying the current icon works for me. Yeah, if a click event from either note could open a note, that would be great (but the modes would have different "drag" behaviours). 

I do like the idea of a key press changing the modes (either as a key press to toggle the modes or holding a key to temporarily change modes -- whichever you'd recommend). Maybe that last one can be added as an additional issue as well.
